packages/connect/src/api/cancelCoinjoinAuthorization.ts

```diff
@@ -0,0 +1,25 @@
+import { AbstractMethod } from '../core/AbstractMethod';
+import { getFirmwareRange } from './common/paramsValidator';
+import { PROTO } from '../constants';
+
+export default class CancelCoinjoinAuthorization extends AbstractMethod<
+    'cancelCoinjoinAuthorization',
+    PROTO.CancelAuthorization
+> {
+    init() {
+        this.firmwareRange = getFirmwareRange(this.name, null, this.firmwareRange);
```

Comment:
add method name to src/data/config next to `authorizeCoinjoin` (capabilities: ['coinjoin']) otherwise it will not work - will return default firmwareRange

Reply:
6ea6346ee4d4a8617d2379363b5a2f621fdb0bb2